platforms/core-runtime/build-configuration/src/main/java/org/gradle/buildconfiguration/tasks/UpdateDaemonJvm.java

```diff
@@ -201,6 +202,17 @@ public List<String> getAvailableVendors() {
         return Arrays.stream(JvmVendor.KnownJvmVendor.values()).filter(e -> e!=JvmVendor.KnownJvmVendor.UNKNOWN).map(Enum::name).collect(Collectors.toList());
     }
 
+    /**
+     * Indicates it the native-image capability is required.
+     *
+     * @since 8.14
+     */
+    @Input
+    @Optional
+    @Incubating
+    @Option(option = "native-image-capable", description = "Indicates if the native-image capability is required.")
+    public abstract Property<Boolean> getNativeImageCapable();
```

Comment:
Would it make sense to extend the `native-image` capability for the `task/project toolchain` ? Allowing to have something like in the DSL

```
java {
    toolchain {
        ...
        nativeImage = true
    }
}
```

Reply:
That's already done, see #32614